README.md

```diff
@@ -1 +1,108 @@
-# libuast
\ No newline at end of file
+# libuast
+
+## Installation
+
+### 1. Build/Install C++ API
+```
+cmake .
+make
+make install
+````
+
+
+### 2.Install python bindings
+
+```
+## move to bindings
+cd bindings
+cmake .
+make
+make install-python
+```
+
+## Usage
+
+### C++
+
+```c++
+#include <iostream>
+#include "uast.h"
+#include "node.h"
+
+void example()
+{
+  auto root = new Node("compilation_unit");
```

Comment:
Even if this is an example, I'd not create this nodes in the heap and have a leak. I'd create them in the stack.